WordPress/Sniffs/VIP/SlowDBQuerySniff.php

```diff
@@ -50,6 +51,10 @@ public function getGroups() {
 	 */
 	public function process_token( $stackPtr ) {
 
+		if ( $this->has_whitelist_comment( 'slow query', $stackPtr ) ) {
+			return;
+		}
+
 		if ( $this->has_whitelist_comment( 'tax_query', $stackPtr ) ) {
 			return;
```

Comment:
The only reason to issue a warning is if we want to remove support for the `tax_query` whitelist comment at some point. It is saying, "Hey, this is deprecated and is going to stop working soon."

Since checking for a whitelist comment does add some overhead (maybe negligible, but still), I'd say that we should probably plan on removing it eventually. Assuming that it is rarely used anyway, the effort to update to the new comment shouldn't be too much for folks. If that assumption is wrong though, then I'd be OK keeping it, I guess. But otherwise, we should add a warning.